static/src/javascripts/projects/commercial/modules/dfp/render-advert.ts

```diff
@@ -18,9 +18,17 @@ import { renderAdvertLabel } from './render-advert-label';
  *
  */
 
+type Callback = {
+	event: {
+		slot: {
+			getOutOfPage: () => unknown;
+		};
+	};
+};
+
 const addClassIfHasClass = (newClassNames: string[]) =>
 	function hasClass(classNames: string[]) {
-		return function onAdvertRendered(_: any, advert: Advert) {
+		return function onAdvertRendered(_: Callback, advert: Advert) {
```

Comment:
Just wondering is this callback the same type as the other 'sizeCallbacks'?

Reply:
I think so as this function gets used by addFluid and addFluid250, although I've removed this now the argument is optional anyway.